client/packages/common/src/ui/components/navigation/Tabs/Tabs.tsx

```diff
@@ -4,23 +4,44 @@ import MuiTabContext from '@mui/lab/TabContext';
 import MuiTabList from '@mui/lab/TabList';
 import { styled } from '@mui/material/styles';
 
+export const Tab = MuiTab;
 export const TabContext = MuiTabContext;
-
 export const TabPanel = MuiTabPanel;
 
-export const TabList = styled(MuiTabList)({
-  minHeight: '42px',
-  maxHeight: '42px',
+const TAB_HEIGHT = '42px';
```

Comment:
this is all clear, right? Yes, it's a bit clunky and I didn't realise that were were fudging the height to allow for the 8px margin. I've kept the current usage and introduced a short tab style as well